code.js

```diff
@@ -1,3 +1,26 @@
 function binarySearch(list, element) {
+    if(list.length == 0) {
+        return -1;
+    }
+    else if(list[0] == element) {
+        return 0;
+    }
+    else {
+        let topList = list.length - 1;
+        let bottomList = 0;
+        while(topList >= bottomList) {
+          let halfList = Math.floor((bottomList + topList)/2)
+          if(element == list[halfList]) {
+            return halfList + 1;
```

Comment:
I don't understand what you're doing here. If the element is at a specific position, you're returning another position?